README.md

```diff
@@ -1,6 +1,28 @@
-# Stellar Tipping bot
+# Starry - A Stellar tipbot for Slack
```

Comment:
The README shouldn't be specific to slack but universal.

We could have a general introduction and then a ##Reddit and a ##Slack section (and maintain that when new adapters arise.

Reply:
Sure thing. Didn't intend to change the README in a commit. In the future I'll prevent this sort of thing by checking out a brand new version that both our forks work from.

Reply:
Fixed.